common/src/test/java/org/apache/servicecomb/toolkit/common/FileUtilsTest.java

```diff
@@ -32,33 +33,44 @@
 
   @Test
   public void createDirectoryTest() {
-    try {
-      FileUtils.createDirectory(null);
-    } catch (IOException e) {
-      assertEquals("Path is null", e.getMessage());
-    }
-
     Path path;
     try {
       path = Files.createTempDirectory("");
       FileUtils.createDirectory(path.toFile().getCanonicalPath());
     } catch (IOException e) {
       fail("Run 'createDirectoryTest' failed and unexpected to catch IOException: " + e.getMessage());
     }
+
+    try {
+      FileUtils.createDirectory(null);
+    } catch (IOException e) {
+      assertEquals("Path is null", e.getMessage());
+      return;
+    }
+
+    fail("Run 'createDirectoryTest' failed, expected to catch IOException but not");
   }
 
   @Test
   public void getFilesGroupByFilenameTest() {
+    boolean succeed = false;
+
     try {
+      succeed = false;
       FileUtils.getFilesGroupByFilename(null);
```

Comment:
It can be changed to like this 
try {
      FileUtils.getFilesGroupByFilename(null);
      Assert.fail("Except RuntimeException Here");
    } catch (RuntimeException e) {
      assertThat(e, is(oops));
    }